src/index.css

```diff
@@ -1,3 +1,212 @@
+  color: #888;
+}
+
+.input-form {
+  display: flex;
+  justify-content: space-between;
+  gap: 2vw;
+}
+
+.input-title {
+  width: 41.5rem;
+  height: 1.5rem;
+}
+
+.input-category {
+  width: 17.813rem;
+  height: 1.9rem;
+}
+
+.input-button {
+  width: 11.5rem;
+  height: 1.9rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  border-color: #0290ff;
+}
+
+.list-container {
+  display: flex;
+  flex-direction: column;
+  gap: 2vh;
+  padding-left: 0;
+}
+
+.box {
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  padding: 2vh 2vw;
+}
+
+.list-left-comp,
+.list-right-comp {
+  list-style: none;
+}
+
+.list-left-category {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.font-roboto {
+  font-family: Roboto Slab, Arial, sans-serif;
+}
+
+.list-left-title {
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.list-left-author {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.list-left-buttons {
+  display: flex;
+  padding-top: 3vh;
+}
+
+.list-left-button,
+.list-left-button2 {
+  width: 4.563rem;
+  height: 1.188rem;
+  margin: 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  padding-right: 1vw;
+}
+
+.list-left-button {
+  padding-right: 1vw;
+}
+
+.list-left-button2 {
+  padding-left: 1vw;
+  border-left: 2px solid #e8e8e8;
+}
+
+.list-left-button-remove {
+  border-top: none;
+  border-bottom: none;
+  border-right: none;
+  padding: 0 1vw;
+  background-color: white;
+  color: #4386bf;
+  border-color: #e8e8e8;
+}
+
+.list-right {
+  display: flex;
+  gap: 3vw;
+  padding-right: 8vw;
+}
+
+.list-right-2 {
+  border-left: 1px solid #e8e8e8;
+  padding-left: 5vw;
+}
+
+.list-right-comp {
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: 300;
+  color: #121212;
+}
+
+.list-right-comp1 {
+  list-style: none;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.list-right-button {
+  width: 8.938rem;
+  height: 1.725rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border-color: #0290ff;
+  color: #fff;
+  margin-top: 2vh;
+}
+
+.list-right-1 {
+  display: flex;
+}
+
+.oval-2 {
+  width: 3.9rem;
+  height: 3.9rem;
+  margin-top: 1vh;
+  border-style: solid;
+  border-width: 5px;
+  color: #379cf6;
+  border-top-color: #ddd;
+  border-image-slice: 1;
+  border-radius: 50%;
+}
+
+.oval-3 {
+  padding-right: 2vw;
+}
+
+.percentage {
+  font-size: 2rem;
+  color: #121212;
+  margin: 0;
+  padding-top: 2vh;
+  padding-left: 2vw;
+}
+
+.completed {
+  margin: 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  color: #121212;
+  padding-left: 2vw;
+}
+
+.under {
+  color: #0290ff;
+  font-weight: 700;
 }
```

Comment:
- [ ] <b>REQUIRED:</b> There are few things we need to fix to make this app look more like the provided mockup design.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_10_2_2021_10-46-59 PM](https://user-images.githubusercontent.com/45217364/135731368-aef79b05-41b7-423b-be12-5d06a1b1d97b.png) |
   - The app is not fullscreen. it's like a card in the center of the screen. kindly fix it.
   - Toolbar needs to be given a white background to distinguished it from the rest of the page, just like in the mockup.
   - There must be a profile icon on the right side of the toolbar. Remember, it doesn't need to be functional.
    
Note: Remember, when a client gives a project, you are expected to deliver the product exactly as they requested it. so kindly try to make sure that your app looks like the provided mockup at least 98%.

Reply:
Fixed